### PR TITLE
Missing layer features

### PR DESCRIPTION
**Pull request by maasencioh**
needs to update the module values

### CODE REVIEW

File: src/modules/module.js [hunk 1]
```diff
@@ -437,6 +437,7 @@ define([
         setLayers: function (layers, blankLayer) {
```
Comment (targos) [inline src/modules/module.js]: you need to pass something here to indicate that a layer needs to be removed

File: src/src/main/grid.js [hunk 2]
```diff
@@ -535,6 +535,76 @@ define([
...
+            var value = form.getValue().sections.layeropts[0].groups.layeropts[0],
+                layer = {name: value.layername[0]};
+
+            if (definition.layers[layer.name] && (Object.keys(definition.layers).length > 1)) {
+                definition.layers[layer.name] = undefined;
+            } else {
+                console.log("layer doesn't exist");
+            }
+
+            setLayers();
```
Comment (maasencioh) [inline src/src/main/grid.js]: setLayers(false, layer.name);

**Comment by maasencioh** [comment]
@targos @stropitek this will fix #271

**Comment by targos** [comment]
thank you, I will try it on Monday

**Comment by maasencioh** [comment]
ok, have a nice weekend

**Comment by targos** [comment]
landed in 89e1cc52a9dfb2a64dcd780a255825e4d00a4b50 b6193f720db8a7008c081026e74bed481e7caa22 7e50be3427dc42decb3a8a6dd7cb220e26c9afd2 c6e6da705556bc58bb1ddb9e4d7e170cdffb299b 946e671062ef31b6a43fbcb04542e10b8849b9eb, thank you !

**Comment by maasencioh** [comment]
Thanks!

**Pull request closed without merging**